### DISCUSSION TITLE
Find the actual Height of the Text

### DISCUSSION THREAD

**Discussion by faizulhaque** [fabricjs/fabric.js#9517] [Q&A] [answered]
Reference: 
![image](https://github.com/fabricjs/fabric.js/assets/836298/5d3aec36-d18d-4507-9d3e-f48537079f4a)

When I try to get the height of the `Text`, it's returning the value inclusive to THIS (red line in the snapshot). 

Is there any way to find/get the true height of the Text?

jsfiddle: https://jsfiddle.net/vm0acp1r/

**Comment by ShaMan123** [comment, marked as answer]
Your repro is using a very old version of fabric
See this: https://jsfiddle.net/a1q0tpyz/1/

```ts
const text = new fabric.Text(...)
const measuringContext = document.createElement('canvas').getContext('2d')
measuringContext.textAlign = text .textAlign
measuringContext.font = text ._getFontDeclaration()
const measurements = measuringContext.measureText(text .text)
const actualHeight = measurements.actualBoundingBoxAscent + measurements.actualBoundingBoxDescent
```

The red line is `actualHeight`
![image](https://github.com/fabricjs/fabric.js/assets/34343793/4024c76f-7f37-4572-a80a-7849d89dc5fe)
